docs/runbooks/deployment-vercel.md

```
@@ -0,0 +1,59 @@
+
+## Step 1: Configure integrations
+
+1) Supabase integration
+
+- Link Supabase project to Vercel project
+- Ensure env vars are injected:
+  - NEXT_PUBLIC_SUPABASE_URL
+  - NEXT_PUBLIC_SUPABASE_ANON_KEY
+  - SUPABASE_SERVICE_ROLE_KEY (server-only)
+
+2) Upstash integration
```

markdownlint-cli2 (0.18.1)

Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)